StripeConnect/StripeConnect/Source/Internal/Webview/MessageHandlers/OnLoadErrorMessageHandler.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  OnLoadErrorMessageHandler.swift
+//  StripeConnect
+//
+//  Created by Chris Mays on 8/9/24.
+//
+
+import Foundation
+
+// Emitted when there is an error loading connect js
+class OnLoadErrorMessageHandler: OnSetterFunctionCalledMessageHandler<OnLoadErrorMessageHandler.Values> {
+    struct Values: Codable, Equatable {
+        let elementTagName: String
+        let error: ErrorValue
+        
+        struct ErrorValue: Codable, Equatable {
+            let type: String
+            let message: String
+        }
+    }
+    
+    init(didReceiveMessage: @escaping (Values) -> Void) {
+        super.init(setter: "setOnLoadError", didReceiveMessage: { values in
+            if let values {
+                didReceiveMessage(values)
+            } else {
+                debugPrint("Did not receive values for onLoad")
```

Comment:
nit: Could we add a TODO to log this to analytics in [MXMOBILE-2491](https://jira.corp.stripe.com/browse/MXMOBILE-2491)?

If we end up fixing the type so this doesn't have to be optional, then it'll automatically get taken care of when we log an analytic if the payload doesn't decode.